src/components/UserDisplay.vue

```diff
@@ -16,14 +15,11 @@
 
 <script>
 export default {
-  name: 'UserDisplay',
-
+ name: 'UserDisplay',
```

Comment:
Your change was removing a space which shouldn't be there
```suggestion
  name: 'UserDisplay',
```